src/NUnitFramework/framework/Internal/TestNameGenerator.cs

```diff
@@ -424,14 +458,9 @@ private static string EscapeControlChar(char c)
 
         private class TestIDFragment : NameFragment
         {
-            public override string GetText(MethodInfo method, object[] args)
-            {
-                return "{i}"; // No id available using MethodInfo
-            }
-
```

Comment:
Can you clarify what this change is?

Reply:
I removed the `MethodInfo` overload when I widened the test name generator from `TestMethod` to any `Test`, including fixtures. It was only a helper overload anyway. I feel like everything is cleaner.